spec/requests/portfolio_items_spec.rb

```diff
@@ -207,7 +207,7 @@
       allow(svc_object).to receive(:process).and_raise(topo_ex)
 
       get "/#{api}/portfolio_items/#{portfolio_item.id}/service_plans", :headers => default_headers
-      expect(response).to have_http_status(:internal_server_error)
+      expect(response).to have_http_status(:not_found)
```

Comment:
@eclarizio I am not sure about this one, we need to account for the topology service not being available for some reason, if we translate that to not found it won't make sense. A not found has to be generated from the topology server that we can forward to the UI or the caller.

Reply:
@mkanoor Well, the spec is testing the fact that a "topo_ex" error is raised. If that happens, that's currently mapped to a "not found" error, so I just fixed the spec to expect the "correct" thing. Maybe we just map it to a 503 (service unavailable)?

Reply:
@mkanoor My bad, for some reason I translated the `rescue_from` that used to be internal server error to not found. Woops! 🤷‍♂.

Changed to a 503.